src/instructlab/model/backends/vllm.py

```diff
@@ -96,25 +96,34 @@ def run_detached(
         http_client: httpx.Client | None = None,
         background: bool = True,
         foreground_allowed: bool = False,
+        max_startup_retries: int = 0,
     ) -> str:
-        try:
-            _, vllm_server_process, api_base = ensure_server(
-                backend=VLLM,
-                api_base=self.api_base,
-                http_client=http_client,
-                host=self.host,
-                port=self.port,
-                background=background,
-                foreground_allowed=foreground_allowed,
-                server_process_func=self.create_server_process,
-                max_startup_attempts=self.max_startup_attempts,
-            )
-            self.process = vllm_server_process or self.process
-            self.api_base = api_base or self.api_base
-        except ServerException as exc:
-            raise exc
-        except SystemExit as exc:
```

Comment:
Is the removal of catching `SystemExit` here because it's redundant and `ilab` would have already crashed if that exception was called?

Reply:
The previous catches weren't doing anything.  They were just catching and immediately raising again.  I assume it was a remnant?